bitnami/redis-cluster/templates/scripts-configmap.yaml

```diff
@@ -11,6 +11,11 @@ metadata:
   {{- end }}
 data:
   ping_readiness_local.sh: |-
+    #!/bin/sh
+    set -e
+
+    REDIS_STATUS_FILE=/tmp/.redis_cluster_check
+
     {{- if .Values.usePasswordFile }}
```

Comment:
Where is the end for this if?

Reply:
I didn't modify that section, simply added the block above:
https://github.com/bitnami/charts/blob/2b122273bad90e2958b695135387448d70bc1cf0/bitnami/redis-cluster/templates/scripts-configmap.yaml#L19-L24